lib/season_stats.rb

```diff
@@ -22,20 +22,30 @@ def games_by_season_id(season_id)
     games_by_id
   end
 
+  def winning_percentage_by_head_coach(season_id)
+    games_by_head_coach = games_by_season_id(season_id).group_by { |game| game.head_coach}
+
+    winning_percentage_by_head_coach = {}
+
+    games_by_head_coach.each do |key, value|
+      winning_games = value.find_all {|value| value.result == "WIN"}
+      if winning_games.length != 0
+        value = ((winning_games.length.to_f / value.length.to_f) * 100).round(2)
+        winning_percentage_by_head_coach[key] = value
```

Comment:
I use similar logic to get win percentages in some team_stats methods, could be good to move to a module once all our code is functional.